docs/src/main/docs/health/02_health_in_k8s.adoc

```diff
@@ -0,0 +1,217 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Kubernetes probes
+:description: Kubernetes probes
+:keywords: helidon, readiness, liveliness, probes, kubernetes
+:kubernetes-probes-url: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes
+
+This document describes usages of the Helidon health check API with Kubernetes.
+
+== About Kubernetes probes
+
+Probes is the term used by Kubernetes to describe health checks for containers
+ (more info link:{kubernetes-probes-url}[here]).
+
+There are 2 types of probes:
```

Comment:
To be _really_ picky, for numbers less than 10 typically spell the number out ("two" instead of "2" in this case). Use digits for 10 and higher. So much fun.

Reply:
Thank you for being picky, I'll fix that.